From: Liang Yan <lyan@suse.com>
Date: Thu, 30 May 2019 16:35:17 -0400
Subject: KVM: s390: kABI Workaround for 'kvm_vcpu_stat'
Patch-mainline: Never, kABI workaround
References: bsc#1119222

Add halt_no_poll_steal to kvm_vcpu_stat. Hide it from the kABI checker.
Related patch:
patches.fixes/KVM-s390-provide-kvm_arch_no_poll-function.patch

Signed-off-by: Liang Yan <lyan@suse.com>
---
 arch/s390/include/asm/kvm_host.h |    2 ++
 1 file changed, 2 insertions(+)

--- a/arch/s390/include/asm/kvm_host.h
+++ b/arch/s390/include/asm/kvm_host.h
@@ -303,7 +303,9 @@ struct kvm_vcpu_stat {
 	u64 halt_successful_poll;
 	u64 halt_attempted_poll;
 	u64 halt_poll_invalid;
+#ifndef __GENKSYMS__
 	u64 halt_no_poll_steal;
+#endif
 	u64 halt_wakeup;
 	u64 instruction_lctl;
 	u64 instruction_lctlg;
